### PR TITLE
Pbf 720

### PR DESCRIPTION
**Pull request by nagavenivn**

### CODE REVIEW

**Comment by nagavenivn** [comment]
@aeng7 @dchauOoyala Could you please review this?

File: src/osmf/lib/HDSPlayer.as [hunk 1]
```diff
@@ -306,7 +308,11 @@ package
           dispatchEvent(new DynamicEvent(DynamicEvent.PAUSED,null));
           break;
         case MediaPlayerState.BUFFERING:
-          dispatchEvent(new DynamicEvent(DynamicEvent.BUFFERING,null));
+          {
```
Comment (aeng7) [inline src/osmf/lib/HDSPlayer.as]: I think the default style of this file does not add these brackets for case statements.

File: src/osmf/js/osmf_flash.js [hunk 2]
```diff
@@ -590,8 +590,8 @@ require("../../../html5-common/js/utils/constants.js");
                                "seekRange": { "begin": seekRange_start, "end": seekRange_end } });
     };
 
-    var raiseCanPlayThrough = function() {
-      newController.notify(newController.EVENTS.BUFFERED);
+    var raiseCanPlayThrough = function(event) {
```
Comment (aeng7) [inline src/osmf/js/osmf_flash.js]: Can we add a JS doc for this? I understand there wasn't one before, but this seems like a good time to add one :)

**Comment by aeng7** [comment]
Can we add some unit tests for these changes?

**Comment by nagavenivn** [comment]
@aeng7 Made changes as per your review comments except adding test cases. (There are no existing unit test cases for this module)

**Comment by nagavenivn** [comment]
@dchauOoyala It seems I need one more approval to get this merged. Could you please take a look?

**Comment by dchauOoyala** [comment]
LGTM

**Pull request merged**